packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -196,19 +196,11 @@ export class WallectConnectSession {
 	private async waitForConfirmation(
 		txId: string
 	): Promise<algosdk.modelsv2.PendingTransactionResponse> {
-		const response = await this.algodClient.status().do();
-		let lastround = response[LAST_ROUND];
-		/*eslint no-constant-condition: ["error", { "checkLoops": false }]*/
-		while (true) {
-			const pendingInfo = await this.algodClient.pendingTransactionInformation(txId).do();
-			if (pendingInfo["pool-error"]) {
-				throw new Error(`Transaction Pool Error: ${pendingInfo["pool-error"] as string}`);
-			}
-			if (pendingInfo[CONFIRMED_ROUND] !== null && pendingInfo[CONFIRMED_ROUND] > 0) {
-				return pendingInfo as algosdk.modelsv2.PendingTransactionResponse;
-			}
-			lastround++;
-			await this.algodClient.statusAfterBlock(lastround).do();
```

Comment:
to confirm: did we do the same for `waitFromConfirmation` in `Algo Operator` in `algob`?

Reply:
I m not sure, because the task I was doing in that only web packages was mentioned
https://www.pivotaltracker.com/story/show/181591414

Reply:
could you please check and create apt task?